solarkraft/src/fetcher/storage.ts

```diff
@@ -167,12 +177,19 @@ export function* yieldListEntriesForContract(
                 )
                 if (ledgerDirent.isFile() && matcher) {
                     const txHash = matcher[1]
-                    // TODO: read the verification result and report it
+                    const filename = join(
+                        ledgerDirent.path,
+                        `entry-${txHash}.json`
+                    )
+                    const contents = JSONbig.parse(
+                        readFileSync(filename, 'utf-8')
+                    )
+                    const status = contents['verification'] ?? 'unverified'
                     yield {
                         contractId,
                         height,
                         txHash,
-                        verification: 'unverified',
+                        verification: status,
```

Comment:
I think it would be more idiomatic to just call the field `verificationStatus` and do this here:

```suggestion
                        verificationStatus,
```

Also from the UX perspective, I think just calling the field `verification` in the JSON can be misleading.